<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dsda-launcher</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LaunchGameButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="argumentText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>357</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(180, 180, 180);

color: #000;
height: 20px;
background-color:rgb(255, 255, 255);
border-radius:3px</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="compLevelSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>55</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maxVisibleItems">
     <number>14</number>
    </property>
    <item>
     <property name="text">
      <string>Default</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Doom v1.2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Doom v1.666</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Doom v1.9 / Doom II</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ultimate Doom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Final Doom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DOSDoom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TASDoom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Boom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LxDoom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MBF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Old PrBoom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Current PrBoom</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>27</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IWAD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="iwadSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>25</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>57</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CompLevel</string>
    </property>
   </widget>
   <widget class="QLabel" name="episodeText">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>100</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Episode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Difficulty</string>
    </property>
   </widget>
   <widget class="QComboBox" name="diffBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="episodeBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>99</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(160, 160, 160);
padding-left: 6px;
color: #000;
height: 20px;
background-color:rgb(255, 255, 255);
border-radius:3px</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="levelBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>129</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(160, 160, 160);
padding-left: 6px;
color: #000;
height: 20px;
background-color:rgb(255, 255, 255);
border-radius:3px</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="levelText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(37, 37, 37);
</string>
    </property>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>335</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#808080;&quot;&gt;Additional Arguments&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Console Log</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>311</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Options</string>
     </attribute>
     <widget class="QCheckBox" name="fastCheck">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>5</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string> Fast Monsters</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="noCheck">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string> No Monsters</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>84</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Resolution&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="noCheck_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>55</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Fullscreen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>84</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Graphics Mode&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="noCheck_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string> Respawn Monsters</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="soloNetCheck">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>5</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Solo Net</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wadsTab">
     <attribute name="title">
      <string>Wads</string>
     </attribute>
     <widget class="QListWidget" name="wadsOnFolder">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(139, 138, 140);
padding-left: 6px;
color: #000;
height: 20px;
background-color:rgb(230, 230, 230);
border-radius:3px</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="plus">
      <property name="geometry">
       <rect>
        <x>274</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pngs/pngs/plus.png);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="minus">
      <property name="geometry">
       <rect>
        <x>274</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pngs/pngs/minus.png);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>272</x>
        <y>90</y>
        <width>26</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="demoTab">
     <attribute name="title">
      <string>Demos</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recordDemo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(180, 180, 180);
padding-left: 6px;
color: #000;
height: 20px;
background-color:rgb(255, 255, 255);
border-radius:3px</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recordDemo_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(180, 180, 180);
padding-left: 6px;
color: #000;
height: 20px;
background-color:rgb(255, 255, 255);
border-radius:3px</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>55</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Playback</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>DSDA</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="toolTip">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(160, 160, 160);
padding-left: 6px;
color: #000;
height: 20px;
background-color:rgb(255, 255, 255);
border-radius:3px</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Don't see any IWAD?     ^&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Press  &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;ctrl + o&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; / &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;cmd + o&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;  and drag your IWADs to the folder that opened&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>295</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2560x1600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2048x1280</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1680x1050</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1650x1050</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1440x900</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1152x720</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024x768</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024x640</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>840x525</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>825x525</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>800x600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>640x480</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>640x400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>320x240</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>320x200</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>295</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 Bit Software</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32 Bit Software</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Open GL</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>216</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>267</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="demoPlayOptions">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>295</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height:10px</string>
    </property>
    <item>
     <property name="text">
      <string>Play Demo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Time Demo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fast Demo</string>
     </property>
    </item>
   </widget>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox</zorder>
   <zorder>LaunchGameButton</zorder>
   <zorder>argumentText</zorder>
   <zorder>compLevelSelect</zorder>
   <zorder>label</zorder>
   <zorder>iwadSelect</zorder>
   <zorder>label_2</zorder>
   <zorder>episodeText</zorder>
   <zorder>label_5</zorder>
   <zorder>diffBox</zorder>
   <zorder>episodeBox</zorder>
   <zorder>levelBox</zorder>
   <zorder>levelText</zorder>
   <zorder>pushButton</zorder>
   <zorder>tabs</zorder>
   <zorder>toolTip</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>demoPlayOptions</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
